topicscanner.go

```diff
@@ -0,0 +1,115 @@
+package kafka
+
+import (
+	"context"
+	"regexp"
+	"sync"
+	"time"
+)
+
+var (
+	defaultScanningIntervalMS = 10000
```

Comment:
what do you think of relaxing the interval to once per minute?

Reply:
FWITW, both the Java client and rdkafka refresh topic metadata every 5 minutes by default.